docs/css-prop.md

```diff
@@ -6,7 +6,38 @@ The primary way to style elements with emotion is the `css` prop. It provides a
 
 ## Get Started
 
-#### Set the jsx pragma at the top of your source file.
+There are 2 ways to get started with the `css` prop.
+
+- [Babel Preset](#babel-preset)
+- [JSX Pragma](#jsx-pragma)
+
+Both methods result in the same compiled code. Choose the one that works for you situation.
```

Comment:
> Choose the one that works for you situation.

This feels kind of pointless, maybe say in a really short way use the babel preset in x cases and use the jsx pragma in y cases(I know it's a lot of repeating but people will skim things a lot)